storybook/stories/Examples/cartesian/Brush/Brush.stories.tsx

```diff
@@ -61,24 +60,6 @@ export const ControlledBrush = {
       </>
     );
   },
-  play: async ({ canvasElement }: { canvasElement: HTMLElement }) => {
-    const user = userEvent.setup();
-    const { getByLabelText } = within(canvasElement);
```

Comment:
Why remove this test? Pretty sure it's still a valid use case no?

Reply:
Oh I see the message now. I don't necessarily agree? Asserting that certain things with in a real browser environment is still important. Maybe not this, but things that measure in the dom for example

Reply:
The test runs on page load, makes the storybook blink twice. The failure is not recorded or displayed. As such it doesn't serve the purpose.

I'm supportive of real browser tests (we will need them to test touch I think) but our storybook setup as is today is not it.